src/Server/Formatter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace LanguageServer\Server;
+
+use LanguageServer\Protocol\TextEdit;
+use LanguageServer\Protocol\Range;
+use LanguageServer\Protocol\Position;
+
+class Formatter
+{
+
+    /**
+     * @param string $content
+     * @param string $uri
+     *
+     * @return TextEdit[]
+     */
+    public function format(string $content, string $uri)
+    {
+        // remove 'file://' prefix
+        $uri = substr($uri, 7);
```

Comment:
I don't think this is the right way to do it - an input of `file:///c:/users/felix/test` would result in `/c:/users/felix/test` which is not a valid path. #31 adds a `path2uri` function, I would propose to add the reverse `uri2path` function (with some tests) https://github.com/MadHed/php-language-server/blob/7b0b9b890cabd3f0c2e1d7f10314ed61f53df110/src/utils.php#L23-L33